packages/bits/spec/components/textbox-number/textbox-number.e2e.ts

```diff
@@ -171,4 +172,38 @@ describe("USERCONTROL textbox-number >", () => {
             expect(await component.getValue()).toBe("1");
         });
     });
+
+    describe("keyboard navigation", () => {
+        beforeEach(async () => {
+            await browser.refresh();
+            component = basic;
+        })
+
+        it("should navigate between elements using TAB", async () => {
+            expect("body")
+                .toEqual(await browser.switchTo().activeElement().getTagName());
```

Comment:
Shouldn't it be vise versa? :) Like, we usually expect the element tag to equal some string. Just curious

Reply:
yes, it's a good point, thank you. I'll fix it